nixos/modules/services/admin/oxidized.nix

```diff
@@ -80,11 +88,70 @@ in
       '';
```

Comment:
Why not default routerDB to the text portion of devicesRouterDB? then we need less conditionals below.